x11-apps/mesa-progs/mesa-progs-8.3.0.ebuild

```diff
@@ -15,12 +15,12 @@ fi
 inherit base autotools toolchain-funcs ${GIT_ECLASS}
 
 DESCRIPTION="Mesa's OpenGL utility and demo programs (glxgears and glxinfo)"
-HOMEPAGE="http://mesa3d.sourceforge.net/"
+HOMEPAGE="https://www.mesa3d.org/ https://mesa.freedesktop.org/"
 if [[ ${PV} == 9999* ]]; then
 	SRC_URI=""
 	KEYWORDS="alpha amd64 arm hppa ia64 ppc ppc64 sparc x86"
 else
-	SRC_URI="ftp://ftp.freedesktop.org/pub/${MY_PN/-//}/${PV}/${MY_P}.tar.bz2"
+	SRC_URI="https://mesa.freedesktop.org/archive/${MY_PN/-//}/${PV}/${MY_P}.tar.bz2"
```

Comment:
I agree this ebuild is ugly... further up in the header of the ebuild, MY_PN is defined in a way which replaces "mesa-progs" with "mesa-demos", which creates a valid (and the expected) URL of https://mesa.freedesktop.org/archive/demos/...

Is the preference to use or not to use ${PN} in SRC_URI and EGIT_REPO_URI? Personally I would prefer that variables such as these not be used, but I note that most ebuilds do it.